route/hophint.go

```diff
@@ -0,0 +1,36 @@
+package route
+
+import "github.com/btcsuite/btcd/btcec"
+
+const (
+	// DefaultFinalCLTVDelta is the default value to be used as the final
+	// CLTV delta for a route if one is unspecified.
+	DefaultFinalCLTVDelta = 9
+)
+
+// HopHint is a routing hint that contains the minimum information of a channel
```

Comment:
Would moving the hop hint struct into the `zpay32` package also address this?

Reply:
Or possible the introduction of a `path` package? So we'd have `path.Find`, etc in there and the objective of the package would be to house less LN specific graph operations. Things like fees and CLTV calculation, etc would still be in the main `routing` package. Already we have a separation between a path and a route, so we wouldn't be forcing any new delineation.

Reply:
Moving in the `zpay32` package works. It isn't too strange either. You could say that a hop hint isn't exclusively a property of an invoice though. Good suggestion, I went for this option.

Wouldn't want to pull in more restructuring like the `path` package now.